docs/node/CATEGORIES/01-blockchain.md

```diff
@@ -237,6 +237,42 @@ interface blockchain {
   getRawMempool (verbose: boolean): Promise<string[] | MempoolTx>
 }
 
+interface MempoolTx {
+  [key: string]: {
+    vsize: BigNumber
+    /**
+     * @deprecated same as vsize. Only returned if defid is started with -deprecatedrpc=size
+     */
+    size: BigNumber
+    weight: BigNumber
+    fee: BigNumber
+    modifiedfee: BigNumber
+    time: BigNumber
+    height: BigNumber
+    descendantcount: BigNumber
+    descendantsize: BigNumber
+    descendantfees: BigNumber
+    ancestorcount: BigNumber
+    ancestorsize: BigNumber
+    ancestorfees: BigNumber
+    wtxid: string
+    fees: {
+      base: BigNumber
+      modified: BigNumber
+      ancestor: BigNumber
+      descendant: BigNumber
+    }
+    depends: string[]
+    spentby: string[]
+    'bip125-replaceable': boolean
+  }
+}
+```
+## getMempoolEntry
+
+Get transaction details in the memory pool using a transaction ID.
+
+```ts title="client.blockchain.getMempoolEntry(txId: string)"
 interface MempoolTx {
```

Comment:
`MempoolTx` interface is already defined above. No need to redefine it.
Instead, add interface for calling

```typescript
interface blockchain {
  getMempoolEntry (txId: string): Promise<MempoolTx>
}
```